### DISCUSSION TITLE
Running playbook failed:-: permission denied

### DISCUSSION THREAD

**Discussion by Banane6077** [semaphoreui/semaphore#1801] [Q&A]
Hello there!

I'm tryin' to use my local Storage as my Repository but i get always the error: "Running playbook failed: stat /home/edv/Playbooks/: permission denied"

![image](https://github.com/ansible-semaphore/semaphore/assets/125888597/7c761d6c-8a1c-4fdd-89e8-b9a0b7b379bf)

![image](https://github.com/ansible-semaphore/semaphore/assets/125888597/3d449acc-06b0-4b0e-90d4-93f00a4f7d60)

What am i doing wrong?
The path is as far as i know correct and i think i dont need a password so i created a Key with "none" Parameters.

Thanks for your help!

**Comment by tboerger** [comment]
The user which is running semaphore is not allowed to access the path. You need to change the ownership.
